app/controllers/api/v0/competitions_controller.rb

```diff
@@ -184,6 +184,14 @@ def update_wcif
     }
   end
 
+  def disable_auto_accept
+    competition = competition_from_params
+    require_can_manage!(competition)
+
+    competition.update!(auto_accept_registrations: false)
+    render json: { status: 'success' }
+  end
```

Comment:
Why is this part of the public `v0` API? Should API users be able to disable auto-accept from the outside world? (FTR: I don't think they should, but perhaps I'm overlooking something)

Reply:
Organizers/delegates should be able to disable auto accept - this is the endpoint that receives the call when "Disable auto accept" is clicked on the Reg Admin page. 

We could make it part of the v1 API instead - but I figured that given it's in the same namespace as the WCIF PATCH and non-public WCIF list it was in the appropriate place?

Reply:
TODO